src/components/PostsList/PostsList.jsx

```diff
@@ -1,37 +1,61 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './PostsList.scss';
 
-export const PostsList = () => (
-  <div className="PostsList">
-    <h2>Posts:</h2>
-
-    <ul className="PostsList__list">
-      <li className="PostsList__item">
-        <div>
-          <b>[User #1]: </b>
-          sunt aut facere repellat provident occaecati excepturi optio
-        </div>
-        <button
-          type="button"
-          className="PostsList__button button"
-        >
-          Close
-        </button>
-      </li>
-
-      <li className="PostsList__item">
-        <div>
-          <b>[User #2]: </b>
-          et ea vero quia laudantium autem
-        </div>
-
-        <button
-          type="button"
-          className="PostsList__button button"
-        >
-          Open
-        </button>
-      </li>
-    </ul>
-  </div>
-);
+import PropTypes from 'prop-types';
+
+export const PostsList = ({ posts, selectedPostId }) => {
+  const [detailedPostId, setDetailedPostId] = useState();
+
+  const checkDetailsOpeness = (postId) => {
+    if (detailedPostId === postId) {
+      selectedPostId();
+      setDetailedPostId();
```

Comment:
selectedPostId doesn't look like a functions name. It's looks like you trying to call a variable, not a function.
what value you're setting to the detailedPostId?